dex/pair/src/ctx_helper.rs

```diff
@@ -201,6 +201,13 @@ pub trait CtxHelper:
         context.set_output_payments(payments);
     }
 
+    fn should_generate_locked_asset(&self) -> bool {
+        let current_epoch = self.blockchain().get_block_epoch();
+        let unlock_epoch = self.unlock_epoch().get();
```

Comment:
we will LOCK in the first epoch only - but UNLOCK happens in like epoch 3. so we need another storage map

Reply:
Done. Also moved into a separate `locking_wrapper` module.